draft-ietf-tls-tls13.md

```diff
@@ -3331,7 +3355,7 @@ useful as a traffic analysis countermeasure.
 
        struct {
            ContentType type;
-           ProtocolVersion legacy_record_version = 0x0301;    /* TLS v1.x */
+           ProtocolVersion legacy_record_version;
```

Comment:
If we believe including a value means the receiver is justified in enforcing it (section 3.7 doesn't say much, but I think I agree that's the more natural read), `ClientHello.legacy_version` probably should get the same treatment. It's a little awkward because this is one of the few places where senders always send one value, but receivers must tolerate multiple.

(And, yeah, enforcing the record-layer version prior to encryption is fraught with peril..)